OpenRA.Game/Sync.cs

```diff
@@ -62,15 +67,13 @@ static void EmitSyncOpcodes(Type type, ILGenerator il)
 				il.Emit(OpCodes.Ldc_I4, 0x555);
 				il.MarkLabel(l);
 			}
-			else if (type.HasAttribute<SyncAttribute>())
```

Comment:
Nevermind, I also got confused, but `SyncAttribute` is required for lint rules and used in `SyncReport` .